cinema/serializers.py

```diff
@@ -1,24 +1,27 @@
 from rest_framework import serializers
 
-from cinema.models import Movie
+from cinema.models import Movie, Actor, Genre, CinemaHall
```

Comment:
You can also use relative imports, IMHO it is better

Reply:
relative imports is a very bad practice)